frankDoc/src/test/resources/doc/testConfigs/testConfiguration.xml

```diff
@@ -13,6 +9,12 @@
         <Forward name="success" path="EXIT" />
       </FixedResultPipe>
       <Pipe className="my.proprietary.pipe"/>
+      <Exit path="EXIT" state="success" code="200"/>
     </Pipeline>
+    <!-- TODO: Receiver should be above Pipeline, but Martijn does not want to correct that on April 22 2021.
+         Fix this by rearranging methods in class Adapter -->
```

Comment:
Als het nodig is om de volgorde van methods in Adapter aan te passen, dan moet je dat doen in deze PR. Niet deze configuratie aanpassen om de test maar groen te krijgen.

Reply:
Gedaan. Ook PipeLine waar hetzelfde punt speelde.